# Patched by Tox for 2
#
# Date: 2025-01-28
#
# Desc: Minor formatting changes && avoid making unnecessary binaries,
# man-pages, and symlinks
#
# Credits: Thank you to bdubbs over at LFS from whom much of this patch was
# taken

diff -Naur b/src/Makefile a/src/Makefile
--- b/src/Makefile	2025-01-28 18:08:15.151184672 -0600
+++ a/src/Makefile	2025-01-28 18:15:32.644191607 -0600
@@ -2,8 +2,8 @@
 # Makefile	Makefile for the systemV init suite.
 #		Targets:   all      compiles everything
 #		           install  installs the binaries (not the scripts)
-#                          clean    cleans up object files
-#			   clobber  really cleans up
+#              clean    cleans up object files
+#			   			 clobber  really cleans up
 #
 # Version:	@(#)Makefile  2.85-13  23-Mar-2004  miquels@cistron.nl
 #
@@ -23,19 +23,15 @@

 # For some known distributions we do not build all programs, otherwise we do.
 BIN	=
-SBIN	= init halt shutdown runlevel killall5 fstab-decode logsave
-USRBIN	= last mesg readbootlog
+SBIN	= init halt shutdown runlevel killall5 fstab-decode

-MAN1	= last.1 lastb.1 mesg.1 readbootlog.1
 MAN5	= initscript.5 inittab.5 initctl.5
-MAN8	= halt.8 init.8 killall5.8 pidof.8 poweroff.8 reboot.8 runlevel.8
+MAN8	= halt.8 init.8 killall5.8 poweroff.8 reboot.8 runlevel.8
 MAN8	+= shutdown.8 telinit.8 fstab-decode.8 logsave.8

 ifeq ($(DISTRO),)
-SBIN	+= sulogin bootlogd
-USRBIN	+= utmpdump wall
-MAN1	+= utmpdump.1 wall.1
-MAN8	+= sulogin.8 bootlogd.8
+SBIN	+= bootlogd
+MAN8	+= bootlogd.8
 endif

 ifeq ($(DISTRO),Debian)
@@ -216,10 +212,6 @@
 		ln -sf halt $(ROOT)$(base_sbindir)/reboot
 		ln -sf halt $(ROOT)$(base_sbindir)/poweroff
 		ln -sf init $(ROOT)$(base_sbindir)/telinit
-		ln -sf $(base_sbindir)/killall5 $(ROOT)$(base_bindir)/pidof
-		if [ ! -f $(ROOT)$(bindir)/lastb ]; then \
-			ln -sf last $(ROOT)$(bindir)/lastb; \
-		fi
 		$(INSTALL_DIR) $(ROOT)$(includedir)/
 		$(INSTALL_DATA) initreq.h $(ROOT)$(includedir)/
 		for lang in  '' $(patsubst ../man/po/%.po,%,$(wildcard ../man/po/??.po)); do \
@@ -246,8 +238,4 @@
 		#
 		# This part is skipped on Debian systems, the
 		# debian.preinst script takes care of it.
-		@if [ ! -p /run/initctl ]; then \
-		 echo "Creating /run/initctl"; \
-		 rm -f /run/initctl; \
-		 mknod -m 600 /run/initctl p; fi
 endif
